Quaver.Shared/Database/Maps/Map.cs

```diff
@@ -383,6 +383,16 @@ public virtual Qua LoadQua(bool checkValidity = false)
             {
                 case MapGame.Quaver:
                     var quaPath = $"{ConfigManager.SongDirectory}/{Directory}/{Path}";
+
+                    try
+                    {
+                        File.OpenText(quaPath);
+                    } catch (FileNotFoundException)
+                    {
+                        qua = new Qua();
+                        break;
+                    }
```

Comment:
Very odd way of handling this. If anything, this error handling should be done when parsing in `Qua.cs`

Reply:
Done with https://github.com/Quaver/Quaver.API/pull/117